test/functional/apps/visualize/_area_chart.js

```diff
@@ -465,5 +467,93 @@ export default function({ getService, getPageObjects }) {
         expect(paths.length).to.eql(numberOfSegments);
       });
     });
+
+    describe('date histogram when no date field', () => {
+      before(async () => {
+        await PageObjects.visualize.loadSavedVisualization('AreaChart [no date field]');
+        await PageObjects.visChart.waitForVisualization();
+
+        log.debug('Click X-axis');
+        await PageObjects.visEditor.clickBucket('X-axis');
+        log.debug('Click Date Histogram');
+        await PageObjects.visEditor.selectAggregation('Date Histogram');
+      });
+
+      it('should show error message for field', async () => {
+        const fieldErrorMessage = await find.byCssSelector(
+          '[data-test-subj="visDefaultEditorField"] + .euiFormErrorText'
+        );
+        const errorMessage = await fieldErrorMessage.getVisibleText();
+        await PageObjects.common.sleep(30000);
```

Comment:
Probably you forgot to remove it? Since wait is happening after `errorMessage` is assigned, it won't add any difference.

Reply:
You're write, it's here by mistake. Thanks for catching this!